test/intl402/NumberFormat/prototype/format/default-value.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018 Ujjwal Sharma. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-number-format-functions
+description: >
+  Tests that the default value for the argument of
+  Intl.NumberFormat.prototype.format (value) is undefined.
+info: |
+  11.1.4Number Format Functions
+
+  3. If value is not provided, let value be undefined.
+---*/
```

Comment:
In the future you'll want to make sure that you include `features: [...]`, but even then, as now, it's ok if they aren't present (I'm working on a tool to automate that, so I like when I get new test material to process with that tool)